Services/service_response.c

```diff
@@ -50,10 +50,8 @@ void service_response_task(void *param) {
 
     if (xQueueReceive(service_queues.response_queue, &packet,
                       NORMAL_TICKS_TO_WAIT) == pdPASS) {
-      cnv8_32(&packet->data[DATA_BYTE], &data);
```

Comment:
Side note: these prints just mess things up when running on hardware so I'm just taking them out for now